frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-participants/experiment-participants.component.ts

```diff
@@ -304,6 +304,9 @@ export class ExperimentParticipantsComponent implements OnInit {
   }
 
   removeMember1(groupIndex: number) {
+    if (groupIndex === 0 && this.members1.controls.at(0).get('type').value === 'All') {
```

Comment:
Are you sure these chained calls will never catch a weird `undefined` or null case? 

`this.members1.controls.at(0).get('type').value === 'All')`
 `this.participantsForm.get('inclusionCriteria').setValue(INCLUSION_CRITERIA.INCLUDE_SPECIFIC)`